src/elm/Page/Community/Objectives.elm

```diff
@@ -244,91 +243,93 @@ viewAction ({ shared } as loggedIn) model objectiveId action =
+                , div [ class "mx-2 mb-2" ]
+                    [ if action.isCompleted then
+                        div [ class "tag bg-green" ] [ text_ "community.actions.completed" ]
 
-                  else if isClosed then
-                    div [ class "tag bg-gray-500 text-red" ] [ text_ "community.actions.closed" ]
+                      else if isClosed then
+                        div [ class "tag bg-gray-500 text-red" ] [ text_ "community.actions.closed" ]
 
-                  else
-                    text ""
+                      else
+                        text ""
+                    ]
                 ]
-            ]
-        , div [ class "flex justify-between items-end py-8 flex-col" ]
-            [ div [ class "w-full" ]
-                [ p [ class "input-label mb-4" ] [ text_ "community.actions.verifiers" ]
-                , if validationType == "AUTOMATIC" then
-                    div [ class "flex items-center" ]
-                        [ p [ class "text-body" ] [ text_ "community.actions.automatic_analyzers" ]
-                        , Icons.exclamation "ml-2"
-                        ]
+            , div [ class "flex flex-wrap justify-between items-end" ]
+                [ div [ class "w-full sm:w-4/5" ]
+                    [ p [ class "input-label mb-4" ] [ text_ "community.actions.verifiers" ]
+                    , if validationType == "AUTOMATIC" then
+                        div [ class "flex items-center" ]
+                            [ p [ class "text-body" ] [ text_ "community.actions.automatic_analyzers" ]
+                            , Icons.exclamation "ml-2"
+                            ]
 
-                  else
-                    div [ class "flex overflow-scroll -mx-2" ]
-                        (List.map
-                            (\u ->
-                                div [ class "mx-2" ]
-                                    [ Profile.view shared loggedIn.accountName u ]
+                      else
+                        div [ class "flex mr-2" ]
+                            (List.map
+                                (\u ->
+                                    div [ class "mr-4" ]
+                                        [ Profile.view shared loggedIn.accountName u ]
+                                )
+                                action.validators
                             )
-                            action.validators
-                        )
-                ]
-            , a
-                [ class "button button-secondary button-sm w-full mt-16"
-                , Route.href (Route.EditAction model.communityId objectiveId action.id)
+                    ]
+                , a
+                    [ class "button button-secondary button-sm w-full sm:w-40 mt-8"
+                    , Route.href (Route.EditAction model.communityId objectiveId action.id)
+                    ]
+                    [ text_ "edit action" ]
```

Comment:
Please, use translations for the texts. Translation variants are stored in `public/translations/*.json` files. If there's no suitable translation then you can add your own in the JSONs appropriate sections.

For this line it could be this:

```elm
text <| t shared.translations "menu.edit" ++ " " ++ t shared.translations "community.actions.title"
```

Or it may be an entry like `community.actions.edit` with the text `Edit action` in every JSON file and the code:

```elm
text (t shared.translations "community.actions.edit")
```

Reply:
Thanks for the tip! I created a label translation and pushed the changes for new review. Tks @amiskov !